ui-tests/src/main/java/org/openmrs/reference/page/AddEditLocationPage.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class AddEditLocationPage extends Page {
+
+    protected static final String PAGE_URL = "adminui/metadata/locations/location.page";
+
+    private static final By SAVE_BUTTON = By.id("save-button");
+    private static final By NAME_FIELD = By.id("name-field");
+    private static final By FIRST_TAG = By.cssSelector("#locationForm > fieldset > table > tbody > tr:nth-child(1) > td:nth-child(2) > input[type=\"checkbox\"]");
```

Comment:
better not to use '>' as even adding a div or span somewhere along the way will break the test.